jest.config.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/en/configuration.html
+ */
+
+module.exports = {
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // Indicates whether the coverage information should be collected while executing the test
+  collectCoverage: true,
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: "coverage",
+
+  // An array of regexp pattern strings used to skip coverage collection
+  coveragePathIgnorePatterns: [
+    "\\\\node_modules\\\\",
+
+    //Ignore index.js because it cannot be reasonably tested, nor is there much value to doing so
+    "index.js"
+  ],
+
+  // Indicates which provider should be used to instrument code for coverage
+  coverageProvider: "v8",
+
+  // A list of reporter names that Jest uses when writing coverage reports
+  // coverageReporters: [
+  //   "json",
+  //   "text",
+  //   "lcov",
+  //   "clover"
+  // ],
+
+  // An object that configures minimum threshold enforcement for coverage results
+  coverageThreshold: {
+    global: {
+      branches: 90,
+      functions: 90,
+      lines: 90,
+    },
+  },
+
+  // Make calling deprecated APIs throw helpful error messages
+  errorOnDeprecated: true,
+
+  // A set of global variables that need to be available in all test environments
+  globals: {},
+
+  // Activates notifications for test results
```

Comment:
Maybe remove these commented configs prior to initial release? Maybe this needs it's own issue